benchmarks/__main__.py

```diff
@@ -2,12 +2,16 @@
 # Run this via 'python -m benchmarks' from the root directory.
 
 from benchmarks.domains import BENCHMARKS
+from benchmarks.persistance import S3ExperimentResultPersistence
 
 
 def main():
```

Comment:
That would be the case, yes. Would you want to store it locally? Might be reasonable so that the result can be used afterwards instead of volatile objects and prints from the results.

Reply:
@AdrianSosic I've added another suggestion but feel free to exchange on further improvements and restructurings :)

Reply:
OK, since we have [another thread](https://github.com/emdgroup/baybe/pull/429#discussion_r1846614901) discussing the architecture, let me close this one here.